health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/attendanceManagementTable.js

```
@@ -1,92 +1,163 @@
-    props?.handlePageChange(page, totalRows);
-  };
+  const handleDaysWorkedChange = (workerId, value) => {
 
-  const handleRowSelect = (event) => {
-    // if(!event?.allSelected && event?.selectedCount >0){
-    //     setIsIntermediate(true);
-    // }
-    props?.onRowSelect(event);
-  };
+    if (value?.target) {
+      value = value?.target?.value;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)